shop/docker/docker-compose-devel.yml

```diff
@@ -37,9 +37,11 @@ services:
       - zed
       - redis
       - elasticsearch
+      - phantomjs
     env_file: ./docker-compose.env
     environment:
       - APPLICATION_ENV=development
+      - ENABLE_OPCACHE=true
```

Comment:
Isn't this the shop scaffold? I just changed it there because you or @twolfy has already changed it in the demoshop